crates/ra_batch/src/vfs_filter.rs

```diff
@@ -1,54 +1,54 @@
-use std::path::PathBuf;

```

Comment:
Did you change the kind of newline in this file? According to git every line in this file has changed.

Reply:
Yeah, it used dos newlines before